packages/oc-webpack/lib/configurator/index.js

```diff
@@ -7,15 +7,20 @@ const path = require('path');
 const webpack = require('webpack');
 
 module.exports = function webpackConfigGenerator(options) {
-  const buildPath = options.buildPath || '/build';
   const production =
     options.production !== undefined ? options.production : 'true';
 
+  const sourceMaps = !production;
+  const devtool = sourceMaps ? '#source-map' : '';
```

Comment:
fun fact: old webpack with infinite-loader would go on with a broken sourcemap, new webpack throws and fails to compile because of it. This is actually something I like more, because in the future, if that would break, it would fail to compile rather than update snapshot with (potentially) broken references 👍